posts/2024-06-04-24.0.0.6-beta.adoc

```diff
@@ -0,0 +1,455 @@
+
+The 24.0.0.6-beta release provides InstantOn support for the link:{url-prefix}/docs/latest/reference/feature/sessionCache-1.0.html[JCache Session Persistence] feature. This feature uses a JCache provider to create a distributed in-memory cache. Distributed session caching is achieved when the server is connected to at least one other server to form a cluster. Open Liberty servers can behave in the following ways in a cluster.
+
+- Client-server model: An Open Liberty server can act as the JCache client and connect to a dedicated JCache server.
+- Peer-to-Peer model: An Open Liberty server can connect with other Open Liberty servers that are also running with the JCache Session Persistence feature and configured to be part of the same cluster.
+
+To enable JCache Session Persistence, the `sessionCache-1.0` feature must be enabled in your `server.xml` file:
+
+[source,xml]
+----
+<feature>sessionCache-1.0</feature>
+----
+
+You can configure the client/server model in the `server.xml`` file, similar to the following example.
+
+[source,xml]
+----
+<library id="InfinispanLib">
+    <fileset dir="${shared.resource.dir}/infinispan" includes="*.jar"/>
+</library>
+<httpSessionCache cacheManagerRef="CacheManager"/>
+<cacheManager id="CacheManager">
+    <properties
+        infinispan.client.hotrod.server_list="infinispan-server:11222"
+        infinispan.client.hotrod.auth_username="sampleUser"
+        infinispan.client.hotrod.auth_password="samplePassword"
+        infinispan.client.hotrod.auth_realm="default"
+        infinispan.client.hotrod.sasl_mechanism="PLAIN"
+        infinispan.client.hotrod.java_serial_whitelist=".*"
+        infinispan.client.hotrod.marshaller=
+            "org.infinispan.commons.marshall.JavaSerializationMarshaller"/>
+    <cachingProvider jCacheLibraryRef="InfinispanLib" />
+</cacheManager>
+----
+
+You can configure the peer-to-peer model in the `server.xml` file, similar to the following example.
+
+[source,xml]
+----
+<library id="JCacheLib">
+    <file name="${shared.resource.dir}/hazelcast/hazelcast.jar"/>
+</library>
+
+<httpSessionCache cacheManagerRef="CacheManager"/>
+
+<cacheManager id="CacheManager" >
+    <cachingProvider jCacheLibraryRef="JCacheLib" />
+</cacheManager>
+----
+
+**Note:**
+In order to provide InstantOn support for the peer-to-peer model by using Infinispan as a JCache Provider, you must use Infinispan 12 or later. You must also enable link:{url-prefiux}/docs/latest/reference/feature/mpReactiveStreams-3.0.html[MicroProfile Reactive Streams 3.0] or later and link:{url-prefix}docs/latest/reference/feature/mpMetrics-4.0.html[MicroProfile Metrics 4.0] or later must in the `server.xml` file, in addition to the JCache Session Persistence feature.

```

Comment:
Must should be removed in the line `or later must in the `server.xml``

Reply:
fixed